haystack/components/converters/azure.py

```diff
@@ -49,7 +51,7 @@ class AzureOCRDocumentConverter:
     ```
     """
 
-    def __init__(
+    def __init__(  # pylint: disable=too-many-positional-arguments
```

Comment:
I know disabling lint here is not appreciated, but if we want to refactor the params it must be done in a different PR.